docs/howto/outside_controller_use.md

```diff
@@ -10,8 +10,8 @@ In ActiveModelSerializers versions 0.10 or later, serializing resources outside
 # Create our resource
 post = Post.create(title: "Sample post", body: "I love Active Model Serializers!")
 
-# Optional options parameters
-options = {}
+# Optional options parameters for both the serializer and instance
+options = {serializer: PostDetailedSerializer, username: 'sample user'}
```

Comment:
Let's mention they're the same options as passed to the controller and link to https://github.com/rails-api/active_model_serializers/blob/master/docs/general/rendering.md#explicit-serializer